### PR TITLE
integrating turborepo

### PR DESCRIPTION
**Pull request by jonty007**
tested build flow from repos
TBC20
TBC721
Swap
Vite Template
NakamotoJS

### CODE REVIEW

File: packages/swap/package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
     "start": "node build/src/main.js",
     "prebuild": "npm run lint",
     "build": "rm -rf build && tsc -p tsconfig.release.json",
+    "build:turbo": "turbo run build",
```
Comment (ClemensLey) [inline packages/swap/package.json:11]: We should rename this into build-turbo

File: package.json [hunk 2]
```diff
@@ -24,6 +24,7 @@
     "email": "clemens@bitcoincomputer.io"
   },
   "private": true,
+  "packageManager": "npm@10.2.4",
```
Comment (ltardivo) [inline package.json:27]: Do we need to specify the exact npm version?

Comment (jonty007) [reply package.json:27]: no we don't need the exact version, but turbo requires it. I added my latest version.

**Review by ClemensLey — APPROVED** [review]
Looks good to me

**Pull request merged**